### PR TITLE
Doc: Supported CUDA Releases

### PR DESCRIPTION
**Pull request by ax3l**
Document tested CUDA releases in README.md as well.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,7 +72,7 @@ This library uses C++11 (or newer when available).
 | std::thread |:white_check_mark:|:white_check_mark:|:white_check_mark:|:white_check_mark:|:white_check_mark:|
 | Boost.Fiber |:white_check_mark:|:white_check_mark:|:white_check_mark:|:white_check_mark:|:white_check_mark:|
 |TBB 2.2+|:white_check_mark:|:white_check_mark:|:white_check_mark:|:white_check_mark:|:white_check_mark:|
-|CUDA 8.0+|:white_check_mark: (nvcc 8.0+)|:white_check_mark: (nvcc 8.0+)|:x:|:white_check_mark: (native)|:x:|
```
Comment (BenjaminW3) [inline README.md]: I am sorry to say, but we are currently not testing gcc 5 with CUDA 8 due to the std::tuple errors. My CMake error messages will make this clearer.

**Comment by ax3l** [comment]
Do you have a suggestion how we should document this?

**Comment by ax3l** [comment]
Please feel free to include this change simply in #528

**Comment by BenjaminW3** [comment]
Yes, I can directly include it in my PR and do the necessary changes.

**Pull request closed without merging**